src/hooks/core/chat/useMoveChatTypeHook.ts

```diff
@@ -36,7 +36,8 @@ const useMoveChatTypeHook = () => {
     const type: {[key: string]: ChannelType} = {
       messaging: isAnonymous ? 'ANON_PM' : 'PM',
       group: 'GROUP',
-      topics: 'TOPIC'
+      topics: 'TOPIC',
+      topicinvitation: 'TOPIC'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The 'topicinvitation' and 'topics' are both mapped to the same 'TOPIC' value. If these two types have different behaviors, this could lead to unexpected results. Please ensure that this is the intended behavior.
```diff
     const type: {[key: string]: ChannelType} = {
       messaging: isAnonymous ? 'ANON_PM' : 'PM',
       group: 'GROUP',
-      topics: 'TOPIC',
-      topicinvitation: 'TOPIC'
+      topics: 'TOPIC',
+      topicinvitation: 'TOPIC_INVITATION' // if 'topicinvitation' should be treated differently
     };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->